ruby/range-cover-vs-include.md

```diff
@@ -0,0 +1,18 @@
+# Range cover? vs include? method
+
+The `cover?` method yields better performance than `include?`,
+however there are subtle differences to when one should be used over other.
```

Comment:
What are you thinking? Providing the benchmark for the users to run, or just saying something like `~ X times faster`?